troposphere/static/js/components/providers/ProvidersMaster.react.js

```diff
@@ -1,72 +1,19 @@
-define(function (require) {
-  "use strict";
+import React from 'react/addons';
+import Backbone from 'backbone';
+import Router from 'react-router';
+import stores from 'stores';
 
-  var React = require('react/addons'),
-    Router = require('react-router'),
-    RouteHandler = Router.RouteHandler,
-    stores = require('stores');
+var RouteHandler = Router.RouteHandler;
```

Comment:
This is totally optionally, but if you want to use `let` here instead of `var` - you can given we are already using ES6 for the `import` and `export` syntax.